encrypted-media/scripts/playback-persistent-license.js

```diff
@@ -0,0 +1,104 @@
+function runTest(config, testname) {
```

Comment:
Yes, it is a strict subset. It tests basic playback for this session type. The duplication is necessary because you can't test for the correct playback event sequence without doing playback.